core/src/group16/project/game/ecs/component/ShieldComponent.kt

```diff
@@ -6,6 +6,12 @@ import com.badlogic.ashley.core.Component
 class ShieldComponent: Component  {
 
     var shield = 0
+    var position = -1
```

Comment:
having a position value in ShieldComponent makes it easier to check position in ShieldSystem without having to check whether the ShieldComponent is left or right, so that's why we've implemented it that way

Reply:
I see, if it is only used for that, then player would suffice.
It is okay, a small fix / code cleanup in the future can be done. :)